### DISCUSSION TITLE
reporte artillery

### DISCUSSION THREAD

**Discussion by ghost** [artilleryio/artillery#1011] [Artillery]
what is the difference between artillery free reports and artillery pro, can you give me an example

**Comment by hassy** [comment]
hi @gerardosantos1 - there is no difference when it comes to reporting, Artillery Pro is meant to be a seamless upgrade that "just works". The difference is that with Artillery Pro you can scale out your tests and run them from hundreds or thousands of workers in your own AWS account.

**Reply by ghost** [reply]
Thank you very much for your answer, it is already clearer to me.
